README.md

```diff
@@ -2,173 +2,160 @@
+         * [Kogito CLI requirements](#kogito-cli-requirements)
+         * [Kogito CLI installation](#kogito-cli-installation)
             * [For Linux](#for-linux)
             * [For Windows](#for-windows)
-            * [Build CLI from source](#build-cli-from-source)
-         * [Deploy a Kogito Service from source with CLI](#deploy-a-kogito-service-from-source-with-cli)
-      * [Development](#development)
-         * [Build Operator](#build-operator)
-         * [Deploy to OpenShift 4.x for development purposes](#deploy-to-openshift-4x-for-development-purposes)
-         * [Running End to End tests](#running-end-to-end-tests)
-            * [Operator SDK](#operator-sdk)
-            * [Kogito CLI](#kogito-cli-1)
-         * [Running Locally](#running-locally)
-      * [Prometheus Integration](#prometheus-integration)
-         * [Prometheus Annotations](#prometheus-annotations)
+            * [Building the Kogito CLI from source](#building-the-kogito-cli-from-source)
+         * [Deploying a Kogito service from source with the Kogito CLI](#deploying-a-kogito-service-from-source-with-the-kogito-cli)
+      * [Kogito Operator development](#kogito-operator-development)
+         * [Building the Kogito Operator](#building-the-kogito-operator)
+         * [Deploying to OpenShift 4.x for development purposes](#deploying-to-openshift-4x-for-development-purposes)
+         * [Running End-to-End (E2E) tests](#running-end-to-end-e2e-tests)
+            * [With the Kogito Operator SDK](#with-the-kogito-operator-sdk)
+            * [With the Kogito CLI](#with-the-kogito-cli)
+         * [Running the Kogito Operator locally](#running-the-kogito-operator-locally)
+      * [Prometheus integration with the Kogito Operator](#prometheus-integration-with-the-kogito-operator)
+         * [Prometheus annotations](#prometheus-annotations)
          * [Prometheus Operator](#prometheus-operator)
-      * [Infinispan Integration](#infinispan-integration)
-      * [Contributing](#contributing)
+      * [Infinispan integration](#infinispan-integration)
+      * [Contributing to the Kogito Operator](#contributing-to-the-kogito-operator)
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Requirements
+## Kogito Operator requirements
 
-- go v1.12
-- [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) v0.11.0
-- ocp 3.11/4.x (you can use [CRC](https://github.com/code-ready/crc) for local deployment)
-- [kogito s2i imagestreams](https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml) installed
+- Go v1.12 is installed.
+- The [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) v0.11.0 is installed.
+- OpenShift 3.11 or 4.x is installed. (You can use [CRC](https://github.com/code-ready/crc) for local deployment.)
+- The [kogito s2i imagestreams](https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml) is installed.
```

Comment:
@ricardozanini This is not a requirement or ? Just after, we put it as "Optional" in the document beacuse Operator does install them ...

Reply:
I’d say to change this to optional since we are installing the images automatically. But should be changed on the JIRA that we introduced this feature. We made a mistake. @sterobin fo you mind adding an “optional” note on this? Thanks!

Reply:
Sure, but I just removed. The section is about "requirements", so we can't have an optional requirement. Plus it's mentioned in the subsequent section where relevant, as you said.